src/realmHydroBody.ttl

```diff
@@ -19,7 +19,7 @@
 @prefix sostc: <http://sweetontology.net/stateChemical/> .
 @prefix sorea: <http://sweetontology.net/realm/> .
 @prefix sorelsc: <http://sweetontology.net/relaSci/> .
-@prefix dct: <http://purl.org/dc/dcmitype/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
```

Comment:
not used, could remove

Reply:
Yes, IIRC from the prefix re-mapping exercise (I believe it was #165), there are unused prefixes scattered throughout.  

Counterpoint: perhaps `dctype` and `dcterm` should be included in every `ttl` file even if not _currently_ used.  As we ramp up adding definitions from multiple sources they will be used for provenance metadata and potentially reification statements.

As you say, these aren't show stoppers; just my 2p.